### DISCUSSION TITLE
Can't deploy Directus to shared server

### DISCUSSION THREAD

**Discussion by federicocappellotto97** [directus/directus#10142] [Archive]
Since 3-4 days i can't deploy directus to cPanel anymore, and this is driving me crazy 😅 The problem is Argon2 has been updated. I tried with 4 different hosting providers, 3 with cPanel and 1 with Plesk, without success.
Does any of you know an hosting to deploy Directus successfully?

**Comment by rijkvanzanten** [comment]
> Does any of you know an hosting to deploy Directus successfully?

See https://docs.directus.io/getting-started/installation/. Any of those will work great 🙂 

Just out of curiosity, what exact hosting providers did you try?

**Reply by federicocappellotto97** [reply]
Two different servers on https://www.a2hosting.com/, one on https://business.aruba.it/home.aspx (this is a custom private server with plesk) and one with webhostingmagic.com. I get the same error everytime :( I'm looking for a shared hosting with cPanel or Plesk
